udata/app.py

```diff
@@ -123,7 +123,23 @@ def default(self, obj):
         return super(UDataJsonEncoder, self).default(obj)
 
 
-def create_app(config='udata.settings.Defaults', override=None):
+WARNING_LOGGERS = 'elasticsearch', 'requests'
+
+
+def init_logging(app):
+    logging.captureWarnings(True)  # Display warnings
+    debug = app.debug or app.config.get('TESTING')
+    log_level = logging.DEBUG if debug else logging.WARNING
+    app.logger.setLevel(log_level)
+    for name in app.config['PLUGINS']:
+        logging.getLogger('udata_{0}'.format(name)).setLevel(log_level)
+    for logger in WARNING_LOGGERS:
```

Comment:
Maybe explain why we do that?

Reply:
:+1:

Reply:
Done (and renamed into `VERBOSE_LOGGERS` to be more explicit

Reply:
👌